tests/Unit/Connector/Sabre/PropFindPluginTest.php

```diff
@@ -65,23 +65,26 @@ protected function setUp(): void {
 		$this->userAgentManager = $this->createMock(UserAgentManager::class);
 		$this->request = $this->createMock(IRequest::class);
 		$this->server = $this->createMock(Server::class);
+		$this->pathCache = $this->createMock(E2EEnabledPathCache::class);
 
-		$this->plugin = new PropFindPlugin($this->rootFolder,
+		$this->plugin = new PropFindPlugin(
+			$this->rootFolder,
 			$this->userSession,
 			$this->userAgentManager,
-			$this->request);
+			$this->request,
+			$this->pathCache
+		);
 	}
 
 	public function testInitialize(): void {
 		$server = $this->createMock(Server::class);
 
-		$server->expects($this->at(0))
-			->method('on')
-			->with('afterMethod:PROPFIND', [$this->plugin, 'checkAccess'], 50);
-
-		$server->expects($this->at(1))
-			->method('on')
-			->with('propFind', [$this->plugin, 'updateProperty'], 105);
+		$server->expects($this->atLeast(2))
+		 ->method('on')
+		 ->willReturnMap([
+		 	['afterMethod:PROPFIND', [$this->plugin, 'checkAccess'], 50],
+		 	['propFind', [$this->plugin, 'updateProperty'], 105],
+		 ]);
```

Comment:
What is returned? If the method returns void I would use withConsecutive instead of willReturnMap.